### ISSUE TITLE
Data Science with Python : Getting Started with Scikit-Learn

### ISSUE THREAD

**Issue by prathimacode-hub** [girlscript/winter-of-contributing#420]
Welcome to 'DSWP' Team, good to see you here

This issue will helps readers in giving all the guidance that one needs to learn about Introduction to Scikit-Learn. Tutorial to Scikit-Learn and how it's applied using sample code.

To get assigned to this issue, add your serial numbers mentioned in the spreadsheet of "Data Science with Python", the approach one would follow and choice you prefer (Documentation, Audio, Video). You can go with all three or any number of options you're interested to work on.

If you had referred any resources, add them up in "DS Resources". Similarly if you had used datasets, include them in "DS Datasets".

Domain : Machine Learning

Mentors Assigned : Nithin / Gauri Bharadwaj

Points to Note :

The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.

"Issue Title" and "PR Title should be the same. Include issue number along with it.

Changes should be made inside the Datascience_With_Python/ directory & Datascience_With_Python branch.

Follow Contributing Guidelines & Code of Conduct before start Contributing.

This issue is only for 'GWOC' contributors of 'DSWP' domain.

All the best. Enjoy your open source journey ahead. 😎

**Comment by priyanshuc4423** [comment]
@prathimacode-hub I would like to be assigned this. I will write some info regarding it such as its workings etc and then explain code regarding it.
sno: 945
type: documentation

**Comment by prathimacode-hub** [comment]
Issue assigned to @priyanshuc4423 for documentation.

**Comment by ShreytaSingh** [comment]
i would like to be assigned for this issue via documentation and video

**Comment by BhaswatiRoy** [comment]
I want to contribute to this issue :                 
Name-Bhaswati Roy   
Domain-Data Science with Python 
Batch-19  
Serial No-787 
Issue Number-420
Contribution Approaches-all modules of scikit learn like preprocessing, train test split, working over each ml algorithms

**Comment by prathimacode-hub** [comment]
Documentation is already taken, if you would like to work on video/audio for this issue, you can let me know. Have you been assigned to any other issue? @ShreytaSingh

**Comment by prathimacode-hub** [comment]
> I want to contribute to this issue :
> Name-Bhaswati Roy
> Domain-Data Science with Python
> Batch-19
> Serial No-787
> Issue Number-420
> Contribution Approaches-all modules of scikit learn like preprocessing, train test split, working over each ml algorithms

An issue is already assigned to you. @BhaswatiRoy

**Comment by BhaswatiRoy** [comment]
> > I want to contribute to this issue :
> > Name-Bhaswati Roy
> > Domain-Data Science with Python
> > Batch-19
> > Serial No-787
> > Issue Number-420
> > Contribution Approaches-all modules of scikit learn like preprocessing, train test split, working over each ml algorithms
> 
> An issue is already assigned to you. @BhaswatiRoy

When I am already assigned any issue then do I need to finish and submit it then only I can apply for other issues?

**Comment by ShreytaSingh** [comment]
> Documentation is already taken, if you would like to work on video/audio for this issue, you can let me know. Have you been assigned to any other issue? @ShreytaSingh

not yet assigned,yes i would like to work in video domain

**Comment by prathimacode-hub** [comment]
> > > I want to contribute to this issue :
> > > Name-Bhaswati Roy
> > > Domain-Data Science with Python
> > > Batch-19
> > > Serial No-787
> > > Issue Number-420
> > > Contribution Approaches-all modules of scikit learn like preprocessing, train test split, working over each ml algorithms
> > 
> > 
> > An issue is already assigned to you. @BhaswatiRoy
> 
> When I am already assigned any issue then do I need to finish and submit it then only I can apply for other issues?

Yes exactly, since it's hard to focus when you have multiple issues assigned. You can get assigned to issues from other domain. But in same domain, 2 issues cannot be assigned. Its one at a time. So that contribution is meaningful and you get some learning experience out of it when you focus to particular issue.

**Comment by 23-24** [comment]
I want to contribute on this issue (documentation)
issue no:420
serial no:500
batch no:15
Approach: introduction, why do we use it, features along with examples

**Comment by BhaswatiRoy** [comment]
> > > > I want to contribute to this issue :
> > > > Name-Bhaswati Roy
> > > > Domain-Data Science with Python
> > > > Batch-19
> > > > Serial No-787
> > > > Issue Number-420
> > > > Contribution Approaches-all modules of scikit learn like preprocessing, train test split, working over each ml algorithms
> > > 
> > > 
> > > An issue is already assigned to you. @BhaswatiRoy
> > 
> > 
> > When I am already assigned any issue then do I need to finish and submit it then only I can apply for other issues?
> 
> Yes exactly, since it's hard to focus when you have multiple issues assigned. You can get assigned to issues from other domain. But in same domain, 2 issues cannot be assigned. Its one at a time. So that contribution is meaningful and you get some learning experience out of it when you focus to particular issue.

I have submitted my pr on last issue now can I work on this issue?

**Comment by prathimacode-hub** [comment]
> > > I want to contribute to this issue :
> > > Name-Bhaswati Roy
> > > Domain-Data Science with Python
> > > Batch-19
> > > Serial No-787
> > > Issue Number-420
> > > Contribution Approaches-all modules of scikit learn like preprocessing, train test split, working over each ml algorithms
> > 
> > 
> > An issue is already assigned to you. @BhaswatiRoy
> 
> When I am already assigned any issue then do I need to finish and submit it then only I can apply for other issues?

You can work on other issue when PR is created. @BhaswatiRoy

**Comment by prathimacode-hub** [comment]
> > Documentation is already taken, if you would like to work on video/audio for this issue, you can let me know. Have you been assigned to any other issue? @ShreytaSingh
> 
> not yet assigned,yes i would like to work in video domain

Issue is assigned to @ShreytaSingh for video contribution

**Comment by BhaswatiRoy** [comment]
> > > > I want to contribute to this issue :
> > > > Name-Bhaswati Roy
> > > > Domain-Data Science with Python
> > > > Batch-19
> > > > Serial No-787
> > > > Issue Number-420
> > > > Contribution Approaches-all modules of scikit learn like preprocessing, train test split, working over each ml algorithms
> > > 
> > > 
> > > An issue is already assigned to you. @BhaswatiRoy
> > 
> > 
> > When I am already assigned any issue then do I need to finish and submit it then only I can apply for other issues?
> 
> You can work on other issue when PR is created. @BhaswatiRoy

yes I have already created my pr and also updated in threads

**Comment by shripaddhopate** [comment]
I would be glad to work on this issue.
Name : Shripad Dhopate
Batch : 21
Type : Documentation , Video
Serial No: 930

**Comment by priyanshuc4423** [comment]
I have my sem now I can't do it right now sorry :(

**Comment by ashish-reddy-20-08** [comment]
@prathimacode-hub ,
I am M. Ashish Reddy. I would like to do AUDIO on this issue.
Batch number: 5
serial number:174.
Approach: A detailed explanation with examples and real life uses

**Comment by prathimacode-hub** [comment]
Issue assigned to @ashish-reddy-20-08 for audio contribution
